[GDL-Next] - Open URL by clicking GDLauncher Icon

DylanBeMe opened the pull request:
do it or don't.. but extra functionality is great. maybe a better coder can make it take you to the gdevs.io site, for me it would load forever, so instead i made it go to the github. this pull request probably won't make it tho :)

<3

DylanBeMe committed:
Add files via upload

DylanBeMe committed:
Add files via upload

fix spelling error

DylanBeMe committed:
Add files via upload

k

DylanBeMe committed:
Fixed a minor spelling mistake

just a little spelling error fixed for my OCD

DylanBeMe committed:
Open github page on clicking the logo on homescreen

I made the logo on the homescreen open your github page, it wouldn't load the gdevs.io website. I also recommend to make the "create account" on the first login open in a new window instead of completely hijacking the launcher.

DylanBeMe committed:
Merge branch 'next' into next

DylanBeMe committed:
Retry making the logo a button

lint is picky when it comes to tabs and spaces wow

DylanBeMe committed:
Reretry making the logo a button

Forgot to add 2 spaces on line 84

DylanBeMe committed:
Rereretry of making the logo a button

should run lint myself but I'm on mobile rn

blarfoon commented on src/app/desktop/components/Navbar.js:
```diff
@@ -80,16 +80,23 @@ const Navbar = () => {
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <input
```
Why is this an input?

DylanBeMe committed:
Fixes that Davide suggested

I did what you suggested; removed target blank, removed input and changed back to img, changed href to gdevs.io instead of github page

DylanBeMe committed:
Fixes that Davide suggested try#2

I'm a failure :D

DylanBeMe committed:
Fixes that Davide suggested try#2

I'm a failure :D

DylanBeMe committed:
Fixes that Davide suggested try#3

I'm still a failure :D

DylanBeMe committed:
Fixes that Davide suggested try#4

I don't like lint

blarfoon committed:
Merge branch 'next' into next

blarfoon committed:
Merge branch 'next' into next
